improver_tests/developer_tools/__init__.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
```

Comment:
Suggest renaming this file to `conftest.py` so that the fixtures in here will be automatically be made available by pytest.

Reply:
Thank you - that will make things much tidier!